support-workers/scripts/update-code-cloudformation.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+./build-cloudformation.sh
+
+aws --region eu-west-1 cloudformation validate-template --template-body file://../cloud-formation/target/cfn.yaml --profile membership
+
+aws cloudformation update-stack \
```

Comment:
This allows us to update the cloudformation stack on code without having to push to github, wait for a build and then deploy via RiffRaff

Reply:
sounds good, I'd be tempted to put it alongsidde the cfn, rather than separately just so people actually notice it and use it